src/elements/content-uploader/ItemAction.js

```diff
@@ -27,6 +27,8 @@ type Props = {
 const ItemAction = ({ status, onClick, intl, isFolder = false }: Props) => {
     let icon = <IconClose />;
     let retry = false;
+    let cancel = false;
```

Comment:
Couldn't these be just a single status that is populated with the `data-resin-target` attribute?